setup.py

```diff
@@ -43,6 +55,7 @@ def read_injector_variable(name):
     with open('README.md') as f:
         long_description = f.read()
 
+
```

Comment:
```suggestion
```

Please no extra whitespace changes.

Reply:
Why is that? It violates PEP-8.

Reply:
Fixing PEP 8 compliance, changing whitespace and otherwise reformatting code doesn't, in my opinion, belong in a pull request dedicated to something else. Those belong in a separate pull request to make viewing the project's history easier and more approachable.

Reply:
OK